.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 3.0
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 3.0
+    - name: Ruby Tests
+      run: ruby bin/test.rb
```

Comment:
> I assume it’s because you’re anticipating the addition of more test files in future

Yes, exactly! Should definitely have read the Minitest documentation 🤦‍♀️  TBH I forgot what I had to work with and assumed I needed to go the most bare-bones route and run tests by requiring all tests files.

Will fix!